boxsdk/object/user.py

```diff
@@ -171,6 +172,45 @@ def get_avatar(self) -> bytes:
         response = self._session.get(url, expect_json_response=False)
         return response.content
 
+    @api_call
+    def upload_avatar(self, image_path: str) -> str:
+        """
+        Upload avatar image to user account. Supported formats are JPG, JPEG and PNG.
+        Maximum allowed file size is 1MB and resilution 1024x1024 pixels.
+
+        :param image_path: Path of the avatar image to upload
+        :return: URLs to existing user avatars that were updated
+        """
+        with open(image_path, 'rb') as image_stream:
+            image_extention = os.path.splitext(image_path)[-1][1:]
```

Comment:
Shouldn't be here `extension` instead of `extention`?

Reply:
nice catch